applications-frontend/src/router.js

```diff
@@ -47,7 +49,17 @@ let router = new Router({
     },
     {
       path: "/configure",
-      component: Configure
+      component: Configure,
+      meta: {
```

Comment:
Can you add `requiresAuth: true` to the Settings route as well while you're at it?